core/src/bmi.jl

```diff
@@ -223,6 +229,13 @@ function create_callbacks(
     tabulated_rating_curve_cb = PresetTimeCallback(tstops, update_tabulated_rating_curve!)
     push!(callbacks, tabulated_rating_curve_cb)
 
+    if config.allocation.use_allocation
+        t_end = seconds_since(config.endtime, config.starttime)
+        tstops = 1:(config.allocation.timestep):t_end
```

Comment:
Do you intentionally start after 1 second rather than at the beginning?

Reply:
No, this is probably due to me not being completely used to 1-based indexing (although I know this doesn't actually have anything to do with that) ^^